fail2ban/tests/clientreadertestcase.py

```diff
@@ -311,9 +315,8 @@ def testConvert(self):
 		# is unreliable
 		self.assertEqual(sorted(filterReader.convert()), sorted(output))
 
-		filterReader = FilterReader(
-			"testcase01", "testcase01", {'maxlines': "5"})
-		filterReader.setBaseDir(TEST_FILES_DIR)
+		filterReader = FilterReader("testcase01", "testcase01", {'maxlines': "5"},
+		  share_config=self.__share_cfg, basedir=TEST_FILES_DIR)
```

Comment:
just another stylistic whining from me,  previously it was more obvious where args where etc, so why not just to extend on that i.e. (just typoing -- could have typos etc, do not cut/paste ;-))

```python
filterReader = FilterReader(
    "testcase01", "testcase01", {'maxlines': "5"},
   share_config=self.__share_cfg, basedir=TEST_FILES_DIR)
```